### PR TITLE
chore: Cleanup some wasm-specific generated code

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs [hunk 1]
```diff
@@ -703,12 +703,6 @@ private void BuildApplicationInitializerBody(IndentedStringBuilder writer, XamlO
 #endif
 			writer.AppendLineIndented($"#endif");
 
-			if (_isWasm)
-			{
-				writer.AppendLineIndented($"// Workaround for https://github.com/dotnet/runtime/issues/44269");
```
Comment (Youssef1313) [inline src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs:708]: Issue is closed long ago so I guess it's no longer relevant.

File: src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs [hunk 2]
```diff
@@ -717,12 +711,6 @@ private void BuildApplicationInitializerBody(IndentedStringBuilder writer, XamlO
 			if (_isUiAutomationMappingEnabled)
 			{
 				writer.AppendLineIndented("global::Uno.UI.FrameworkElementHelper.IsUiAutomationMappingEnabled = true;");
-
-				if (_isWasm)
-				{
-					// When automation mapping is enabled, remove the element ID from the ToString so test screenshots stay the same.
-					writer.AppendLineIndented("global::Uno.UI.FeatureConfiguration.UIElement.RenderToStringWithId = false;");
```
Comment (Youssef1313) [inline src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs:724]: As this was a workaround specifically for tests, this is now in SamplesApp itself.

**Review by Youssef1313 — COMMENTED** [review]
@jeromelaban Is the reason why the tests fail here clear? I can't see why

File: src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs (hunk 2 above)
Comment (jeromelaban) [reply src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs:724]: Apps won't have this parameter by default. It's best to keep it here.

**Comment by jeromelaban** [comment]
Here's the failure:

```
System.InvalidOperationException: Unable to find Uno.UI.Runtime.WebAssembly in the loaded assemblies\n   at Windows.UI.Xaml.HtmlElementHelper.GetUnoUIRuntimeWebAssembly()\n   at Windows.UI.Xaml.HtmlElementHelper..cctor()\n   Exception_EndOfInnerExceptionStack\n   at Windows.UI.Xaml.UIElement..ctor(String htmlTag, Boolean isSvg)\n   at Windows.UI.Xaml.FrameworkElement..ctor(String htmlTag, Boolean isSvg)\n   at Windows.UI.Xaml.FrameworkElement..ctor(String htmlTag)\n   at Windows.UI.Xaml.Controls.Control..ctor(String htmlTag)\n   at Windows.UI.Xaml.Controls.Control..ctor()\n   at Windows.UI.Xaml.Controls.ContentControl..ctor()\n   at Windows.UI.Xaml.Controls.Frame..ctor()\n   at SamplesApp.App.InitializeFrame(String arguments)\n   at SamplesApp.App.OnLaunched(LaunchActivatedEventArgs e)\n   at Windows.UI.Xaml.Application.Initialize()\n   at Uno.UI.Dispatching.CoreDispatcher.InvokeOperationSafe(UIAsyncOperation operation)"
```

and I would not be suprised if it were related to the removal of `writer.AppendLineIndented($"typeof(global::Uno.UI.Runtime.WebAssembly.HtmlElementAttribute).GetHashCode();");`

**Comment by jeromelaban** [comment]
@Youssef1313 I don't think this behavior was altered in .NET 8, but until we revisit, let's close this PR. We can still open a new issue as needed.

**Pull request closed without merging**